lib/inject/bower-files.js

```diff
@@ -11,26 +11,135 @@ var combined = require('combined-stream'),
     fs       = require('fs');
 
 /**
+ * @param {string|RegExp} pattern Matches the extension of bower files
+ * @param {object?} options Options for <code>gulp.src()</code> stream
+ */
+module.exports = function src(pattern, options) {
```

Comment:
Refactored file somewhat. Since only the `src` feature was being used I have simplified the API to only that function.